### PR TITLE
Add ansible cli options --ask-vault-password and --vault-pass-file

### PR DESCRIPTION
**Pull request by ivog74**
##### SUMMARY
The ansible-playbook command provides 2 options to specify a vault password:
--ask-vault-pass
--vault-password-file

One option use 'pass', the other uses 'password'. I propose to use either pass or password for both option. For now I added the following options:
-- ask-vault-password
--vault-pass-file

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
ansible cli

##### ADDITIONAL INFORMATION

before:
```bash
ansible-playbook --ask-vault-pass ....
ansible-playbook --vault-password-file=<filename> ....
```

after:
```bash
ansible-playbook --ask-vault-password ....
ansible-playbook --vault-password-file=<filename> ....
```

or

```bash
ansible-playbook --ask-vault-pass ....
ansible-playbook --vault-pass-file=<filename> ....
```

### CODE REVIEW

File: lib/ansible/cli/arguments/option_helpers.py [hunk 1]
```diff
@@ -365,5 +365,9 @@ def add_vault_options(parser):
     base_group = parser.add_mutually_exclusive_group()
     base_group.add_argument('--ask-vault-pass', default=C.DEFAULT_ASK_VAULT_PASS, dest='ask_vault_pass', action='store_true',
```
Comment (samdoran) [inline lib/ansible/cli/arguments/option_helpers.py]:
```suggestion
     base_group.add_argument('--ask-vault-pass', '--ask-vault-password', default=C.DEFAULT_ASK_VAULT_PASS, dest='ask_vault_pass', action='store_true',
```

File: lib/ansible/cli/arguments/option_helpers.py [hunk 2]
```diff
@@ -365,5 +365,9 @@ def add_vault_options(parser):
...
                             help='ask for vault password')
+    base_group.add_argument('--ask-vault-password', default=C.DEFAULT_ASK_VAULT_PASS, dest='ask_vault_pass', action='store_true',
+                            help='ask for vault password')
     base_group.add_argument('--vault-password-file', default=[], dest='vault_password_files',
```
Comment (samdoran) [inline lib/ansible/cli/arguments/option_helpers.py]:
```suggestion
     base_group.add_argument('--vault-pass-file', '--vault-password-file', default=[], dest='vault_password_files',
```

**Review by samdoran — CHANGES_REQUESTED** [review]
Rather than adding additional calls to `add_argument()`, just add additional parameters to the existing `add_argument()` calls.

Also, please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) and add integration tests to `test/integration/targets/vault`. See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

**Review by bcoca — APPROVED** [review]

**Review by samdoran — CHANGES_REQUESTED** [review]
Please add integration tests.

**Comment by Akasurde** [comment]
@ivog74 Are you still working on this? Thanks.

**Comment by ivog74** [comment]
> @ivog74 Are you still working on this? Thanks.

Hi @Akasurde, as far as I'm concerned this PR is finished. Do I still need to add something to the PR ?

**Comment by Akasurde** [comment]
> Hi @Akasurde, as far as I'm concerned this PR is finished. Do I still need to add something to the PR ?
Could you please add integration test for this change [here](https://github.com/ansible/ansible/blob/devel/test/integration/targets/vault/runme.sh)?

**Review by samdoran — APPROVED** [review]

**Pull request merged**